### PR TITLE
feat(content): Remnant -  Remnant Peregrine event adjustment

### PR DESCRIPTION
**Pull request by Zitchas**
**Content (Artwork / Missions / Jobs)**

## Summary
Changes:
- Removes Peregrine from shipyard
- Adds Bellows to outfitter
- Adds dialog mentioning that it is being assigned to specific captains currently, but will be available for requisition in the future.

## Background
Basically, the current availability doesn't fit the Remnant's industrial capacity. It'd be nice to have it eventually filter into a "can buy one per month" and then eventually to full normal availability. Barring that, though, it fits better to have it not available for sale at all, and the player is simply limited to the one they are given until later in Chapter 3. It just doesn't make sense to have it suddenly available for sale in bulk.

## Artwork Checklist
 ~~- [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified~~
 ~~- [X] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}~~
 ~~- [X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~~

### CODE REVIEW

File: data/remnant/remnant 2 cognizance.txt [hunk 1]
```diff
@@ -1379,7 +1379,7 @@ mission "Remnant: Cognizance 32"
...
-			`	Taely walks right up to its nose, and the ship crouches down so she can put her hand on it. "Captain <first>, I would like to introduce you to the Peregrine. It is the first new capital warship design we have built in over a century, and is the result of decades of work on my part as well as many others. Hopefully, it is the first of many new ships that are successful."`
+			`	Taely walks right up to its nose, and the ship crouches down so she can put her hand on it. "Captain <first>, I would like to introduce you to the Peregrine. It is the first new capital warship design we have built in over a century, and is the result of decades of work on my part as well as many others. Hopefully, it is the first of many new ships that are successful." She pauses before continuing. "Initially they will be assigned directly to specific captains, but they will eventually be available for requisition normally."`
```
Comment (warp-core) [inline data/remnant/remnant 2 cognizance.txt:1382]:
```suggestion
			`	Taely walks right up to its nose, and the ship crouches down so she can put her hand on it. "Captain <first>, I would like to introduce you to the Peregrine. It is the first new capital warship design we have built in over a century, and is the result of decades of work on my part as well as many others. Hopefully, it is the first of many new ships that are successful." She pauses before continuing. "Initially, they will be assigned directly to specific captains, but they will eventually be available for requisition normally."`
```

```suggestion
			`	Taely walks right up to its nose, and the ship crouches down so she can put her hand on it. "Captain <first>, I would like to introduce you to the Peregrine. It is the first new capital warship design we have built in over a century, and is the result of decades of work on my part as well as many others. Hopefully, it is the first of many new ships that are successful." She pauses before continuing. "For now, they will be assigned directly to specific captains, but they will eventually be available for requisition normally."`
```

Personally, I would've written the latter, but I think there should be a comma regardless.

**Review by warp-core — APPROVED** [review]

**Comment by warp-core** [comment]
Do the fleet variants reflect where their production capacity for these ships is?

**Comment by Zitchas** [comment]
I'm going to take another look through the Peregrine's appearance in fleets, but probably in a separate PR. This change with removing it from sale was the change I really want to get into 0.9.15; whereas adjusting how or when the Peregrine appears in fleets can wait for 0.9.16. (although it would be nice to have in 0.9.15, it's not what I'd consider essential)

**Review by Hurleveur — COMMENTED** [review]
How dare you remove this ship from me??
*closes PR* (I'm kidding)

Seriously though, could you give just *one* Peregrine to the player? (also you could sell one per month; via a spaceport mission)

**Comment by Zitchas** [comment]
> How dare you remove this ship from me?? _closes PR_ (I'm kidding)
> 
> Seriously though, could you give just _one_ Peregrine to the player? (also you could sell one per month; via a spaceport mission)

Yes, the player gets a Peregrine. This mission already gives the player a Peregrine at the appropriate point. Just now that's the *only* Peregrine the player can get at this point, instead of throwing open the floodgates to whole packs of Peregrines instantly.

**Review by Hurleveur — APPROVED** [review]
Looks good, small change but one that makes sense.

I wasn't sure at which point you get one of these bad boys, 1 will have to do for now.

**Review by W1zrad — COMMENTED** [review]
Looks fine to me. How come you replaced it with the afterburner? I don't recall that being unlocked with the story.

**Comment by warp-core** [comment]
The afterburner is supposed to become available by this point. It's on the Merganser and Peregrine, but it doesn't currently seem to be added to the outfitter. It would perhaps make sense to actually add that when the Merganser is added to the shipyard.

**Comment by Zitchas** [comment]
Warp-core has the way of it. My logic is that while it is initially only available on the Merganser, by the time the player is given a Peregrine the Bellows has reached sufficient production to be added to the general outfitting availability. It's not a perfect representation of something filtering into increased availability (the player could theoretically buy infinite mergansers just to pull the Bellows off and use them on their own fleet), but it's about as good as we get in ES.

**Review by W1zrad — APPROVED** [review]
Sounds good to me!

**Pull request merged**